docs/xxopenj9commandlineenv.md

```diff
@@ -0,0 +1,40 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]OpenJ9CommandLineEnv
+
+This option controls whether the VM captures the command line in the environment variable `OPENJ9_JAVA_COMMAND_LINE`.
```

Comment:
Same comment about generic "command line" as before. Also, I think we could usefully add the info from the env var topic after the first sentence plus we could use some clarification of what enabled/disable means for the behavior (it's pretty obvious but worth clarifying just in case). Eg: "When this option is enabled, the VM captures the command that was used to start the VM in the environment variable `OPENJ9_JAVA_COMMAND_LINE`. You can later use this environment variable to find the parameters that were set when the VM started."